### PR TITLE
PWGHF: update saved variables in toXiPi workflow

### PR DESCRIPTION
**Pull request by ZFederica**
Update saved variables in toXiPi workflow:
- remove chi2AtPCACandidate in candidateCreator and treeCreator
- add PV chi2 in treeCreator

### CODE REVIEW

File: PWGHF/TableProducer/treeCreatorToXiPi.cxx [hunk 1]
```diff
@@ -260,6 +263,9 @@ struct HfTreeCreatorToXiPi {
       candidate.decLenV0(),
       candidate.errorDecayLengthCharmBaryon(),
       candidate.errorDecayLengthXYCharmBaryon(),
+      candidate.impactParCascXY() / candidate.errImpactParCascXY(),
+      candidate.impactParPiFromCharmBaryonXY() / candidate.errImpactParPiFromCharmBaryonXY(),
+      candidate.decLenCharmBaryon() / candidate.errorDecayLengthCharmBaryon(),
```
Comment (fcatalan92) [inline PWGHF/TableProducer/treeCreatorToXiPi.cxx:268]: There are no dynamic columns providing directly the normalised values?

**Review by fcatalan92 — APPROVED** [review]
Hi @ZFederica, I have only a small question. In any case, the PR is fine.

File: PWGHF/TableProducer/treeCreatorToXiPi.cxx (hunk 1 above)
Comment (ZFederica) [reply PWGHF/TableProducer/treeCreatorToXiPi.cxx:268]: Hi @fcatalan92, in the tree creators I never saw dynamic column being used. I could use such columns for the candidate creator table, but there I do not store these variables yet (I do not use them neither in the candidateCreator nor in the candidateSelector). This was my idea, is it fine?

Comment (fcatalan92) [reply PWGHF/TableProducer/treeCreatorToXiPi.cxx:268]: Hi Federica, I'm referring to the part where you fill the table. The point is that you can define the operations you are doing here as dynamic columns of the `HfCandToXiPi` table, and then call `candidate.normDecLenXXX()`.
BTW, looking at the table produced by the candidate creator, https://github.com/AliceO2Group/O2Physics/blob/5e5d04beba8ea82e35ba673b9fdf039acfa9870b/PWGHF/DataModel/CandidateReconstructionTables.h#L1087 I see you do not use dynamic or expression column at all. Maybe you could revise it in the future.

**Pull request merged**